app/controllers/invites_controller.rb

```diff
@@ -20,8 +20,8 @@ def accept
       set_website(@invite.website)
       # it will redirect to errors url because we set the website for the user
       after_login_redirect
-    else
-      # it will redirect to create website wizard because we don't sent it
+    elsif @invite.email.casecmp(current_user.email) == 1
+      # it will redirect to website wizard url because emails don't match
       after_login_redirect
```

Comment:
The idea is that you are making a check to see if the emails are different when it's clear they are because we are making a check in the above elsif to see if they are the same. So as a logical consequence it's clear that this else will only happen if the email is different and you are not logged in. !!